### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by HarrisMBrad-cell**
For clarity on what we are working twords.

### CODE REVIEW

**Review by HarrisMBrad-cell — COMMENTED** [review]
Updated for needed examples and clarity...

**Review by HarrisMBrad — APPROVED** [review]
For the Landing Page needs

**Review by HarrisMBrad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,56 +2,56 @@
 
 ## For Motal
  - Required MVP (Game modeled and extensible)
-A: Modal Box
+A: Modal Box | Page
```
Comment (HarrisMBrad-cell) [inline README.md:5]: yes

File: README.md [hunk 2]
```diff
@@ -2,56 +2,56 @@
 
 ## For Motal
  - Required MVP (Game modeled and extensible)
-A: Modal Box
+A: Modal Box | Page 
 
 ## Q: How will I manage my time to complete this project?
-A:  Timeboxed: 4hrs, 1month/2 week sprint cycle & auto gen issue tracking on triggered in Jira.
+A:  Timeboxed: 1hr, 2month/2 week sprint cycle & auto gen issue tracking on triggered in Jira & Circle CI (dev-branch)
```
Comment (HarrisMBrad-cell) [inline README.md:8]: yes

File: README.md [hunk 3]
```diff
@@ -2,56 +2,56 @@
...
 Oh Young Ebon Yon 
 & Aaa That’s Right Productions
 RE: mobile-based or a web application
 
 ### tic-tac-to-round-five
 
-
+Developers should be able to add new features here via pull requests.
```
Comment (HarrisMBrad-cell) [inline README.md:15]: yes

File: README.md [hunk 4]
```diff
@@ -2,56 +2,56 @@
...
 
-
+Developers should be able to add new features here via pull requests. 
 This is a project Tic-Tac-Toe(Game 5) for learning or practicing lifting state up in React.
 Tic-Tac-Toe App with moves marker. Good for all ages and stages of development in on going project. My tribute to Dr. Faulkner
 From round four: https://github.com/HarrisMBrad/tttr-four
 
 ## Just Fork or Clone
-
+ - Intructions on how to git started in your local environment.
```
Comment (HarrisMBrad-cell) [inline README.md:21]: yes

File: README.md [hunk 5]
```diff
@@ -2,56 +2,56 @@
...
 From round four: https://github.com/HarrisMBrad/tttr-four
 
 ## Just Fork or Clone
-
+ - Intructions on how to git started in your local environment. 
 Lets you play tic-tac-toe,
 Indicates when a player has won the game,
 Stores a game’s history as a game progresses,
-Allows players to review a game’s history and see previous versions of a game’s board.
+Allows players to review a game’s history and see previous versions of a game’s board... ready for new features.
```
Comment (HarrisMBrad-cell) [inline README.md:25]: yes

File: README.md [hunk 6]
```diff
@@ -2,56 +2,56 @@
...
-
+ - Intructions on how to git started in your local environment. 
 Lets you play tic-tac-toe,
 Indicates when a player has won the game,
 Stores a game’s history as a game progresses,
-Allows players to review a game’s history and see previous versions of a game’s board.
+Allows players to review a game’s history and see previous versions of a game’s board... ready for new features. 
 
 # To Devs: Use markdown in a new dev-branch`ofchoice`;
+ - This is expected from developers working on this project.
```
Comment (HarrisMBrad-cell) [inline README.md:28]: yes

File: README.md [hunk 7]
```diff
@@ -2,56 +2,56 @@
...
 
-Here's a numbered list:
+Here's a numbered list: new features
 
-1.  first item
-2.  second item
-3.  third item
+1.  first new feature will be updated UI Color schemes to match needed for Page Needs. 
+2.  second new feature will be a Help Page.
+3.  third new feature will be Vendors List for Marketing Needs.
```
Comment (HarrisMBrad-cell) [inline README.md:57]: yes

Comment (HarrisMBrad-cell) [inline README.md:57]: yes

File: README.md [hunk 8]
```diff
@@ -2,56 +2,56 @@
...
 
 ## Add Items
 
-Here's a numbered list:
+Here's a numbered list: new features
 
-1.  first item
-2.  second item
-3.  third item
+1.  first new feature will be updated UI Color schemes to match needed for Page Needs.
```
Comment (HarrisMBrad-cell) [inline README.md:55]: yes

File: README.md [hunk 9]
```diff
@@ -2,56 +2,56 @@
...
 
-Use 3 dashes for an em-dash. Use 2 dashes for ranges (ex., "it's all
+> Use 3 dashes for an em-dash. Use 2 dashes for ranges (ex., "it's all
 in chapters 12--14"). Three dots ... will be converted to an ellipsis.
 Unicode is supported. ☺
 
 ## Add Items
 
-Here's a numbered list:
+Here's a numbered list: new features
```
Comment (HarrisMBrad-cell) [inline README.md:53]: yes

File: README.md [hunk 10]
```diff
@@ -2,56 +2,56 @@
...
 
 > Block quotes are
 > written like so.
->
+
 > They can span multiple paragraphs,
 > if you like.
 
-Use 3 dashes for an em-dash. Use 2 dashes for ranges (ex., "it's all
+> Use 3 dashes for an em-dash. Use 2 dashes for ranges (ex., "it's all
```
Comment (HarrisMBrad-cell) [inline README.md:47]: yes

File: README.md [hunk 11]
```diff
@@ -2,56 +2,56 @@
...
+- the other third new feature
 
-Note that --- not considering the asterisk --- the actual text
+> Note that --- not considering the asterisk --- the actual text
 content starts at 4-columns in.
 
 > Block quotes are
 > written like so.
->
+
```
Comment (HarrisMBrad-cell) [inline README.md:43]: maybe

**Review by HarrisMBrad-cell — COMMENTED** [review]
yes

**Review by HarrisMBrad — APPROVED** [review]
Codofi React

**Pull request closed without merging**